docs/zh-CN/docs/get-starting/get-starting.md

```diff
@@ -25,7 +25,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# 快速开始
+# 快速开始likx测试
```

Comment:
Please do not arbitrarily modify the correct document, this will cause your PR will not be merged

Reply:
Sorry my tool is abnormal, how can I revoke it?